production/technical_userguide.md

```diff
@@ -113,6 +112,9 @@ The examples below include [cURL](https://curl.haxx.se){:target="_blank"} comman
 
 ## Examples
 
+### Generate Client Code
+An OpenAPI v3 JSON schema file, which represents the service capabilities of the API, can be accessed at [https://sandbox.bcda.cms.gov/api/v1/swagger/openapi.json](https://sandbox.bcda.cms.gov/api/v1/swagger/openapi.json).  This file can be used to generate client code to interact with BCDA.  There are various tools that can be used to generate client code from an OpenAPI v3 JSON schema file; one of which is Swagger Editor ([https://editor.swagger.io/](https://editor.swagger.io/)), which can generate code in dozens of different programming languages and also produces a `README` describing usage of the newly generated client code.
```

Comment:
Probably should do `{:target="_blank"}` to ensure all links open in other tabs, which is consistent with the rest of the site.